obitamockbank100.catalog.obita.tech/resources/deploymentGuide.md

```diff
@@ -0,0 +1,42 @@
+# Deployment Guide: Open Banking Solution
+This deployment guide is designed to help customers deploy our Open Banking solution in various environments. Our solution provides a comprehensive set of APIs and services to enable banks to extend their functionality and comply with Open Banking requirements. This document focuses on the deployment options available for the Open Banking solution.
+
+## DeploymentArchitecture
```

Comment:
Please use a simple deployment architecture diagram which has only required components. this architecture is there for the solution deployed on Obita APK

Reply:
Also ensure the deployment guide doesn't contain anything abou open banking solutions